### PR TITLE
Convert MIAA and Postgres deploy from Dialog to NotebookWizard

### LINKED ISSUE #12577: Move Arc instance deployments to use Notebook wizards
It's a better deployment experience for now and lets users configure settings prior to deploying.

Should also make Notebook wizards support "Script" and "Run" buttons - script will generate notebook but not execute while run would script and execute notebook.

[AB#3777](https://mssqltools.visualstudio.com/dea8be32-0532-4a8f-a2f4-89271d8b1cc5/_workitems/edit/3777)

### PR DESCRIPTION
**Pull request by ranasaria**
**_Description:_**
This PR changes the MIAA and Postgres instance deploy experience from the dialog tab to the notebook wizard.

Here are screenshots of the new wizards:

**_SQL Miaa_**
image

**_Postgres Instance_**
image

This PR fixes #12577 & #12599 

**_Testing_**: End to end testing of MIAA and Postgres instance deployment scenarios

### CODE REVIEW

File: extensions/arc/notebooks/arcDeployment/deploy.arc.data.controller.ipynb [hunk 1]
```diff
@@ -47,7 +47,7 @@
                 "|Tools|Description|Installation|\n",
                 "|---|---|---|\n",
                 "|kubectl | Command-line tool for monitoring the underlying Kubernetes cluster | [Installation](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-using-native-package-management) |\n",
-                "|azdata | Command-line tool for installing and managing resources in an Azure Arc cluster |[Installation](https://github.com/microsoft/Azure-data-services-on-Azure-Arc/blob/master/scenarios/001-install-client-tools.md) |"
+                "|Azure Data CLI (azdata) | Command-line tool for installing and managing resources in an Azure Arc cluster |[Installation](https://docs.microsoft.com/en-us/sql/azdata/install/deploy-install-azdata?view=sql-server-ver15) |"
```
Comment (Charles-Gagnon) [inline extensions/arc/notebooks/arcDeployment/deploy.arc.data.controller.ipynb:50]: https://docs.microsoft.com/sql/azdata/install/deploy-install-azdata

File: extensions/arc/notebooks/arcDeployment/deploy.arc.data.controller.ipynb [hunk 2]
```diff
@@ -56,7 +56,7 @@
         {
             "cell_type": "markdown",
             "source": [
-                "### **Setup**"
+                "### **Setup and Check prerequisites**"
```
Comment (Charles-Gagnon) [inline extensions/arc/notebooks/arcDeployment/deploy.arc.data.controller.ipynb:59]: Capitalize Prerequisites

File: extensions/arc/notebooks/arcDeployment/deploy.postgres.existing.arc.ipynb [hunk 3]
```diff
@@ -41,7 +41,21 @@
         {
             "cell_type": "markdown",
             "source": [
-                "### **Check prerequisites**"
+                "### **Prerequisites** \n",
+                "Ensure the following tools are installed and added to PATH before proceeding.\n",
+                " \n",
+                "|Tools|Description|Installation|\n",
+                "|---|---|---|\n",
+                "|Azure Data CLI (azdata) | Command-line tool for installing and managing resources in an Azure Arc cluster |[Installation](https://docs.microsoft.com/en-us/sql/azdata/install/deploy-install-azdata?view=sql-server-ver15) |"
```
Comment (Charles-Gagnon) [inline extensions/arc/notebooks/arcDeployment/deploy.postgres.existing.arc.ipynb:49]: https://docs.microsoft.com/sql/azdata/install/deploy-install-azdata

File: extensions/arc/notebooks/arcDeployment/deploy.postgres.existing.arc.ipynb [hunk 4]
```diff
@@ -41,7 +41,21 @@
...
+            ],
+            "metadata": {
+                "azdata_cell_guid": "20fe3985-a01e-461c-bce0-235f7606cc3c"
+            }
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "### **Setup and Check prerequisites**"
```
Comment (Charles-Gagnon) [inline extensions/arc/notebooks/arcDeployment/deploy.postgres.existing.arc.ipynb:58]: capitalize

File: extensions/arc/notebooks/arcDeployment/deploy.sql.existing.arc.ipynb [hunk 5]
```diff
@@ -41,7 +41,21 @@
         {
             "cell_type": "markdown",
             "source": [
-                "### **Check prerequisites**"
+                "### **Prerequisites** \n",
+                "Ensure the following tools are installed and added to PATH before proceeding.\n",
+                " \n",
+                "|Tools|Description|Installation|\n",
+                "|---|---|---|\n",
+                "|Azure Data CLI (azdata) | Command-line tool for installing and managing resources in an Azure Arc cluster |[Installation](https://docs.microsoft.com/en-us/sql/azdata/install/deploy-install-azdata?view=sql-server-ver15) |"
```
Comment (Charles-Gagnon) [inline extensions/arc/notebooks/arcDeployment/deploy.sql.existing.arc.ipynb:49]: https://docs.microsoft.com/sql/azdata/install/deploy-install-azdata

File: extensions/arc/notebooks/arcDeployment/deploy.sql.existing.arc.ipynb [hunk 6]
```diff
@@ -41,7 +41,21 @@
...
+            ],
+            "metadata": {
+                "azdata_cell_guid": "d1c8258e-9efd-4380-a48c-cd675423ed2f"
+            }
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "### **Setup and Check prerequisites**"
```
Comment (Charles-Gagnon) [inline extensions/arc/notebooks/arcDeployment/deploy.sql.existing.arc.ipynb:58]: capitalize

**Review by Charles-Gagnon — APPROVED** [review]

**Comment by Charles-Gagnon** [comment]
I assume you'll be following up in a separate PR to add the "Script + Run" button to these wizards?

**Review by alanrenmsft — APPROVED** [review]

**Comment by ranasaria** [comment]
> I assume you'll be following up in a separate PR to add the "Script + Run" button to these wizards?

Yes.

**Pull request merged**